integration/app/test-demo/src/pages/VideoFilterTestApp.tsx

```diff
@@ -0,0 +1,115 @@
+  ControlBar,
+  useMeetingManager,
+  useVideoInputs,
+  useLocalVideo,
+  useMeetingStatus,
+  MeetingStatus,
+  VideoTileGrid,
+  FormField,
+  Select,
+  VideoInputBackgroundBlurControl,
+} from 'amazon-chime-sdk-component-library-react';
+import MeetingInfo from '../components/MeetingInfo';
+import MeetingLeaveControl from '../components/MeetingLeaveControl';
+import MeetingForm from '../containers/MeetingForm';
+
+export const VideoFilterTestApp: React.FC = () => {
+  const defaultOptions: BackgroundBlurOptions = {
+    blurStrength: 20,
+    filterCPUUtilization: 20,
+    reportingPeriodMillis: 1000,
+  };
+  const [backgroundBlurOptions, setBackgroundBlurOptions] = useState<BackgroundBlurOptions>(defaultOptions);
+
+  const setBlurStrength = (newBlurStrength: number): void => {
+    console.log(`Setting blur strength to ${newBlurStrength} - should re-initialize the bg blur provider`);
+    const options = {...backgroundBlurOptions, blurStrength : newBlurStrength};
+    setBackgroundBlurOptions(options);
+  }
+
+  return(
+    <div style={{ display: 'flex', flexDirection: 'column', width: '70%', margin: 'auto' }}>
+      <BackgroundBlurProvider options={backgroundBlurOptions}>
+        <MeetingProvider {...{logLevel: LogLevel.INFO}}>
+          <h3 data-testid='app-name'>Video Filter Test</h3>
+          <Meeting onBlurStrengthChanged={(newBlurStrength) => setBlurStrength(newBlurStrength)}/>
+        </MeetingProvider>
+      </BackgroundBlurProvider>
+    </div>
+    )
+};
+
+interface Props {
+  onBlurStrengthChanged: (blurStrength: number) => void;
+}
+
+const Meeting: React.FC<Props> = ({ onBlurStrengthChanged }) => {
+  const { isBackgroundBlurSupported, createBackgroundBlurDevice } = useBackgroundBlur();
+  const meetingManager = useMeetingManager();
+  const { selectedDevice } = useVideoInputs();
+  const { toggleVideo } = useLocalVideo();
+  const meetingStatus = useMeetingStatus();
+  const [ blurStrength, setBlurStrength ] = useState<string>("20");
+
+  useEffect(() => {
+    const addBackgroundBlur = async () => {
+      if (
+        isBackgroundBlurSupported &&
+        meetingStatus === MeetingStatus.Succeeded
+      ) {
+        console.log("Creating background blur device called");
+        const chosenVideoTransformDevice = await createBackgroundBlurDevice(selectedDevice);
+        console.log(chosenVideoTransformDevice);
+        await meetingManager.selectVideoInputDevice(chosenVideoTransformDevice);
+        toggleVideo();
+      }
+    };
+    addBackgroundBlur();
+  }, [
+    isBackgroundBlurSupported,
+    meetingStatus,
+  ]);
+
+  const updateBlurStrength = (e: ChangeEvent<HTMLSelectElement>) => {
+    e.preventDefault();
+    setBlurStrength(e.target.value);
+    console.log('Updated to blurStrength:' + e.target.value);
+    const newBlurStrength = parseInt(e.target.value);
+    onBlurStrengthChanged(newBlurStrength);
+  }
+
+  return (
+    meetingStatus !== MeetingStatus.Succeeded ?
+    <MeetingForm /> :
```

Comment:
Could we remove this condition? I like the idea that putting the meeting form and actual meeting content in separate "pages". But this means we might need to update the test/check steps in `BaseTestPage.js` which should be shared in different test apps.

For example, `checkIfAttendeeHasLeftMeeting` step checks if the `meetingStatus` is "Left". But with this condition, the `meetingStatus` is not displayed on the page after clicking the `leaveMeeting` button.

Reply:
It can be a little confusing when we have the Meeting Video Grid and Meeting Form open at the same time, it should typically be one or the other. Is there some use case where we'd need the Meeting Video Grid when were not in a meeting, or a Meeting Form when we're in a meeting?

If you just want the meeting Info, I'll keep that meeting info somewhere on the page at all times.